distribution/pom.xml

```diff
@@ -39,6 +39,131 @@
         <artifactId>exec-maven-plugin</artifactId>
         <version>1.1.1</version>
         <executions>
+          <!-- Various Scalas -->
+          <execution>
```

Comment:
not getting `viennacl(S)` with mvn clean install -Pdistribution (though maybe this is not needed?  looks like its is only in the `package` arg.  Doesn't seem like an issue- but raising point.

Reply:
Checked again with `mvn clean package -Pdistribution`:
```
mvn clean package -Pdistribution 
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.apache.mahout:mahout-math:jar:0.13.1-SNAPSHOT
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.apache.mahout:mahout-hdfs:jar:0.13.1-SNAPSHOT
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.apache.mahout:mahout-mr:jar:0.13.1-SNAPSHOT
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.apache.mahout:mahout-math-scala_2.10:jar:0.13.1-SNAPSHOT
[WARNING] 'artifactId' contains an expression but should be a constant. @ org.apache.mahout:mahout-math-scala_${scala.compat.version}:[unknown-version], /home/andy/sandbox/mahout_cuda/math-scala/pom.xml, line 30, column 15
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.apache.mahout:mahout-spark_2.10:jar:0.13.1-SNAPSHOT
[WARNING] 'artifactId' contains an expression but should be a constant. @ org.apache.mahout:mahout-spark_${scala.compat.version}:[unknown-version], /home/andy/sandbox/mahout_cuda/spark/pom.xml, line 30, column 15
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.apache.mahout:mahout-h2o_2.10:jar:0.13.1-SNAPSHOT
[WARNING] 'artifactId' contains an expression but should be a constant. @ org.apache.mahout:mahout-h2o_${scala.compat.version}:[unknown-version], /home/andy/sandbox/mahout_cuda/h2o/pom.xml, line 30, column 15
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] Mahout Build Tools
[INFO] Apache Mahout
[INFO] Mahout Math
[INFO] Mahout HDFS
[INFO] Mahout Map-Reduce
[INFO] Mahout Integration
[INFO] Mahout Examples
[INFO] Mahout Math Scala bindings
[INFO] Mahout Spark bindings
[INFO] Mahout H2O backend
[INFO] Mahout Release Package
```  
No `viennacl`

Reply:
all tests pass for me with ```mvn clean install -Dscala.compat.version=2.11 -Dscala.version=2.11.6 -Pviennacl -DskipTests && mvn test -Dscala.compat.version=2.11 -Dscala.version=2.11.6 -Pviennacl  
```

Reply:
`mvn clean install -Pviennacl,viennacl-omp,distribution`

There are executions to pick up multiple scala versions of both in `Pdistribution`

Reply:
👍